app/views/new_document/choose_document_type.html.erb

```diff
@@ -1,11 +1,11 @@
-<% content_for :browser_title, @supertype.label %>
+<% content_for :browser_title, I18n.t("supertypes.#{@supertype.id}.label") %>
```

Comment:
We can actually just do `t`, instead of `I18n.t` in the views.

Reply:
nice one, thanks :)